src/bitcoin-chainstate.cpp

```diff
@@ -113,7 +123,7 @@ int main(int argc, char* argv[])
 
 
     // SETUP: Chainstate
-    auto chainparams = CChainParams::Main();
+    auto chainparams = CChainParams::SigNet({});
```

Comment:
Why?

Reply:
Just not done testing yet and SigNet is easier.